hw/ip/otp_ctrl/doc/_index.md

```diff
@@ -25,30 +301,495 @@ The following table lists the main parameters used throughout the OTP controller
 
 Localparam     | Default (Max)         | Top Earlgrey | Description
 ---------------|-----------------------|--------------|---------------
+**TODO add**   |                       |              |
+
 
 ### Signals
 
 {{< hwcfg "hw/ip/otp_ctrl/data/otp_ctrl.hjson" >}}
 
-Signal                  | Direction        | Type                      | Description
-------------------------|------------------|---------------------------|---------------
+The table below lists other OTP controller signals.
+**TODO: this will likely have to be adjusted as the implementation progresses**
+
+Signal                   | Direction        | Type                        | Description
+-------------------------|------------------|-----------------------------|---------------
+`otp_csrng_o`            | `output`         | `otp_csrng_req_t`           | Entropy request to [CSRNG]({{< relref "hw/ip/csrng/doc" >}}).
+`otp_csrng_i`            | `input`          | `otp_csrng_rsp_t`           | Entropy acknowledgement from [CSRNG]({{< relref "hw/ip/csrng/doc" >}}).
+`pwr_otp_init_i`         | `input`          | `pwrmgr_pkg::pwr_otp_req_t` | Initialization request coming from power manager  **TODO: link to power manager docs**
+`pwr_otp_init_o`         | `output`         | `pwrmgr_pkg::pwr_otp_rsp_t` | Initialization response going to power manager  **TODO: link to power manager docs**
+`otp_pwr_state_o`        | `output`         | `otp_pwr_state_t`           | Current OTP state, used by power manager in order to determine whether a shutdown needs to be aborted due to an ongoing OTP operation.
+`lc_otp_program_i`       | `input`          | `lc_otp_program_req_t`      | Life cycle state transition request. See also **TODO: link to life cycle docs**
+`lc_otp_program_o`       | `output`         | `lc_otp_program_rsp_t`      | Life cycle state transition request. See also **TODO: link to life cycle docs**
+`lc_provision_en_i`      | `input`          | `lifecycle_pkg::lc_tx_t`    | Provision enable qualifier coming from life cycle controller. This signal enables read / write access to the RMA_TOKEN and CREATOR_ROOT_KEY_SHARE0 and CREATOR_ROOT_KEY_SHARE1. **TODO: link to life cycle docs**
+`lc_test_en_i`           | `input`          | `lifecycle_pkg::lc_tx_t`    | Test enable qualifier coming from from life cycle controller. This signals enables the TL-UL access port to the proprietary OTP IP. **TODO: link to life cycle docs**
+`otp_lc_data_o`          | `output`         | `otp_lc_data_t`             | Life cycle state output holding the current life cycle state and the value of the transition counter.
+`otp_keymgr_key_o`       | `output`         | `keymgr_key_t`              | Key output to the key manager holding CREATOR_ROOT_KEY_SHARE0 and CREATOR_ROOT_KEY_SHARE1.
+`otp_flash_key_o`        | `output`         | `keymgr_key_rsp_t`          | Key output to the key flash controller holding FLASH_DATA_KEY and FLASH_ADDR_KEY.
```

Comment:
i can't completely remember now..but did you want a req/ack interface for the key consumers?

Reply:
yeah that's the idea for the ephemeral scrambling keys, and the flash key yes. I have only transcribed here what I already have in the hjson, so the interface description for FLASH needs to be updated (I'll do that right now). 

for the CREATOR_ROOT_KEY I was thinking to just output the value once valid (i.e., something like KEY + valid signal), since that comes straight out of the partition and does not go through the scrambling datapath...